### ISSUE TITLE
Calling restive from https

### ISSUE THREAD

**Issue by Lionel09** [obihill/restive.js#29]
Using restive over "https" protocol causes this error at first call

```
Mixed Content: The page at 'https://mywebsite.com/' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://www.restive.io/ping/ua.php'. This request has been blocked; the content must be served over HTTPS.
```

Possible to update the script to make an https endpoint call?

**Comment by obihill** [comment]
A few questions to help in getting a better idea of your setup:

1. What version of Restive.JS are you using?
2. What browser and operating system are you testing this on?

Thanks.

**Comment by Lionel09** [comment]
1. v1.3.1
2. FF/Ubuntu

I think this problem is easy to fix, just update protocol "http" to "https" in https://github.com/obihill/restive.js/blob/master/restive.js#L3163

**Comment by obihill** [comment]
Thanks for the info.

There was an issue earlier on with the regex; it was built for proxy browsers but Linux systems were actually executing the XHR script wrongly.

The issue was fixed after version 1.3.1 though (as you can probably tell in the link you provided). Could you use the latest build i.e. 1.3.5?

Let me know if you still have issues after that.

**Comment by Lionel09** [comment]
Ok thanks a lot
